apps/condo/domains/common/components/LayoutContext.tsx

```diff
@@ -62,20 +65,21 @@ export const LayoutContextProvider: React.FC<LayoutContextProviderProps> = (prop
     } = useTopNotificationsHook(props.serviceProblemsAlert)
 
     const toggleCollapsed = () => {
-        localStorage && localStorage.setItem('isCollapsed', String(!isCollapsed))
-        setIsCollapsed(!isCollapsed)
+        const newValue = !isCollapsed
```

Comment:
Best to add a comment that you save this state to cookie to prevent flick on hydration